crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -1,27 +1,39 @@
 //! Entity handling types.
 //!
-//! In Bevy ECS, there is no monolithic data structure for an entity. Instead, the [`Entity`]
-//! `struct` is just a *generational index* (a combination of an ID and a generation). Then,
-//! the `Entity` maps to the specific [`Component`s](crate::component::Component). This way,
-//! entities can have meaningful data attached to it. This is a fundamental design choice
-//! that has been taken to enhance performance and usability.
+//! In the ECS paradigm, the **entity** is an abstract and elementary concept. It is closely related
+//! to [components]. An entity exclusively owns zero or more component instances, all of different
+//! types, and can dynamically acquire or lose them over its lifetime.
+//!
+//! See [`Entity`] to learn more.
+//!
+//! [components]: crate::component::Component
 //!
 //! # Usage
 //!
-//! Here are links to the methods used to perform common operations
-//! involving entities:
+//! Operations involving entities and their components are performed either from a system by submitting commands, or from the outside (or from an exclusive system), by directly using [`World`] methods:
+//!
+//! |Operation|Command|Method|
```

Comment:
I love the use of the table here. We should remember this trick.

Reply:
This should be remembered among other formatting tips for #4754.